Tiltfile

```diff
@@ -61,13 +62,25 @@ def deploy_capi():
         extra_args = settings.get("extra_args")
         if extra_args.get("core"):
             core_extra_args = extra_args.get("core")
-            if core_extra_args:
```

Comment:
Why did we remove this test? Just to simplify things?

Reply:
It's based off of @willie-yao's comments since on line 63 we're already testing the value of `extra_args.get("core")`, so it would be redundant to check again.